tests/test_zarr_avg_merger.py

```diff
@@ -289,13 +289,17 @@ class ZarrAvgMergerTests(unittest.TestCase):
     def test_zarr_avg_merger_patches(self, arguments, patch_locations, expected):
         if "compressor" in arguments:
             if arguments["compressor"] != "default":
-                arguments["compressor"] = zarr.codec_registry[arguments["compressor"].lower()]()
+                arguments["compressor"] = numcodecs.registry.codec_registry[arguments["compressor"].lower()]()
         if "value_compressor" in arguments:
             if arguments["value_compressor"] != "default":
-                arguments["value_compressor"] = zarr.codec_registry[arguments["value_compressor"].lower()]()
+                arguments["value_compressor"] = numcodecs.registry.codec_registry[
```

Comment:
I'd suggest putting `codec_reg = numcodecs.registry.codec_registry` at the top of this method then using that variable in the three places the registry is needed just to shorten the lines a bit.

Reply:
good point, updated it in the latest commit. Thanks!